### DISCUSSION TITLE
Flask Deployment Problem

### DISCUSSION THREAD

**Discussion by XGavinp** [vercel/community#6745] [Help]
### Summary

When I am going to deploy my flask project on Vercel when it is deployment is going on I am getting this error

22:03:31.981 | Installing required dependencies...
22:05:43.531 | Build Completed in /vercel/output [2m]
22:05:44.466 | Deploying outputs...
22:08:53.139 | Failed to process build result for "main.py". Data: {"type":"Lambda"}.
22:08:53.143 | Error: data is too long
22:08:55.459

### Example

https://github.com/XGavinp/Vision-Semantics-Image-Captioner-Project

### Steps to Reproduce

1. Select Repo
2. Click Deploy
3. https://vision-semantics-git-main-gavin-pereiras-projects.vercel.app/
![Screenshot 2024-05-06 221927](https://github.com/vercel/community/assets/111137522/840c7d08-87ea-424c-9da8-99ecd2060f72)

**Comment by versecafe** [comment]
@XGavinp What's the total size of the project you are attempting to deploy? the Lambda functions have a maximum size of 250MB.

**Reply by XGavinp** [reply]
Ohk never knew it had a size limit.

**Reply by g1gasad** [reply]
okay I had a same problem, ty
